### PR TITLE
fix lots of little issues reported by codacy

### PR DESCRIPTION
**Pull request by jasoncalabrese**

### CODE REVIEW

File: lib/plugins/boluswizardpreview.js [hunk 1]
```diff
@@ -121,7 +119,7 @@ function init() {
   bwp.updateVisualisation = function updateVisualisation (sbx) {
 
     var results = sbx.properties.bwp;
-    if (results == undefined) return;
```
Comment (jasoncalabrese) [inline lib/plugins/boluswizardpreview.js]: Codacy Issue found: [Expected '===' and instead saw '=='.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/761330305/issues/source?bid=2452379&fileBranchId=2452987#l122)

File: static/js/client.js [hunk 2]
```diff
@@ -963,9 +969,9 @@ var app = {}, browserSettings = {}, browserStorage = $.localStorage;
         })
         .attr('fill', function (d) { return d.color; })
         .style('opacity', function (d) { return highlightBrushPoints(d) })
-        .attr('stroke-width', function (d) {if (d.type == 'mbg') return 2; else return 0; })
-        .attr('stroke', function (d) { return 'white'; })
-        .attr('r', function(d) { if (d.type == 'mbg') return 4; else return 2;});
+        .attr('stroke-width', function (d) { return d.type == 'mbg' ? 2 : 0; })
```
Comment (jasoncalabrese) [inline static/js/client.js]: Codacy Issue found: [Expected '===' and instead saw '=='.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/761330330/issues/source?bid=2452379&fileBranchId=2452987#l972)

File: lib/plugins/boluswizardpreview.js [hunk 3]
```diff
@@ -57,9 +55,9 @@ function init() {
   bwp.checkNotifications = function checkNotifications (sbx) {
 
     var results = sbx.properties.bwp;
-    if (results == undefined) return;
```
Comment (jasoncalabrese) [inline lib/plugins/boluswizardpreview.js]: Codacy Issue found: [Expected '===' and instead saw '=='.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/761330305/issues/source?bid=2452379&fileBranchId=2452987#l58)

File: lib/sandbox.js [hunk 4]
```diff
@@ -161,8 +162,7 @@ function init ( ) {
   }
 
   function unitsLabel ( ) {
-    if (sbx.units == 'mmol') return 'mmol/L';
-    return 'mg/dl';
```
Comment (jasoncalabrese) [inline lib/sandbox.js]: Codacy Issue found: [Expected '===' and instead saw '=='.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/761330323/issues/source?bid=2452379&fileBranchId=2452987#l165)

File: static/js/client.js [hunk 5]
```diff
@@ -493,7 +498,7 @@ var app = {}, browserSettings = {}, browserStorage = $.localStorage;
         })
         .attr('fill', function (d) { return d.color; })
         .attr('opacity', function (d) { return futureOpacity(d.date.getTime() - latestSGV.x); })
-        .attr('stroke-width', function (d) { if (d.type == 'mbg') return 2; else return 0; })
+        .attr('stroke-width', function (d) { return d.type == 'mbg' ? 2 : 0; })
```
Comment (jasoncalabrese) [inline static/js/client.js]: Codacy Issue found: [Expected '===' and instead saw '=='.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/761330330/issues/source?bid=2452379&fileBranchId=2452987#l501)

File: static/js/client.js [hunk 6]
```diff
@@ -963,9 +969,9 @@ var app = {}, browserSettings = {}, browserStorage = $.localStorage;
...
         .style('opacity', function (d) { return highlightBrushPoints(d) })
-        .attr('stroke-width', function (d) {if (d.type == 'mbg') return 2; else return 0; })
-        .attr('stroke', function (d) { return 'white'; })
-        .attr('r', function(d) { if (d.type == 'mbg') return 4; else return 2;});
+        .attr('stroke-width', function (d) { return d.type == 'mbg' ? 2 : 0; })
+        .attr('stroke', function ( ) { return 'white'; })
+        .attr('r', function (d) { return d.type == 'mbg' ? 4 : 2; });
 
       if (badData.length > 0) {
         console.warn("Bad Data: isNaN(sgv)", badData);
```
Comment (jasoncalabrese) [inline static/js/client.js]: Codacy Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/761330330/issues/source?bid=2452379&fileBranchId=2452987#l977)

**Comment by Fokko** [comment]
Looks good! Many changes in numbers, but most of it is just changing the double-quote to single quote, etc. Got to love Codacy!

**Comment by jasoncalabrese** [comment]
Trying to get the easy cleanup out of the way, then keeping things clean will be easier

File: lib/plugins/boluswizardpreview.js (hunk 1 above)
Comment (jasoncalabrese) [inline lib/plugins/boluswizardpreview.js]: Codacy Issue found: [Expected '===' and instead saw '=='.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/761367804/issues/source?bid=2452379&fileBranchId=2452987#l122)

File: static/js/client.js [hunk 7]
```diff
@@ -963,12 +969,12 @@ var app = {}, browserSettings = {}, browserStorage = $.localStorage;
         })
         .attr('fill', function (d) { return d.color; })
         .style('opacity', function (d) { return highlightBrushPoints(d) })
-        .attr('stroke-width', function (d) {if (d.type == 'mbg') return 2; else return 0; })
-        .attr('stroke', function (d) { return 'white'; })
-        .attr('r', function(d) { if (d.type == 'mbg') return 4; else return 2;});
+        .attr('stroke-width', function (d) { return d.type == 'mbg' ? 2 : 0; })
```
Comment (jasoncalabrese) [inline static/js/client.js]: Codacy Issue found: [Expected '===' and instead saw '=='.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/761367829/issues/source?bid=2452379&fileBranchId=2452987#l972)

File: lib/plugins/boluswizardpreview.js (hunk 3 above)
Comment (jasoncalabrese) [inline lib/plugins/boluswizardpreview.js]: Codacy Issue found: [Expected '===' and instead saw '=='.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/761367804/issues/source?bid=2452379&fileBranchId=2452987#l58)

File: static/js/client.js [hunk 8]
```diff
@@ -1199,8 +1210,8 @@ var app = {}, browserSettings = {}, browserStorage = $.localStorage;
       });
     var arcs = treatmentDots.append('path')
```
Comment (jasoncalabrese) [inline static/js/client.js]: Codacy Issue found: ['arcs' is defined but never used.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/761367829/issues/source?bid=2452379&fileBranchId=2452987#l1211)

File: lib/sandbox.js (hunk 4 above)
Comment (jasoncalabrese) [inline lib/sandbox.js]: Codacy Issue found: [Expected '===' and instead saw '=='.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/761367822/issues/source?bid=2452379&fileBranchId=2452987#l165)

File: static/js/client.js (hunk 7 above)
Comment (jasoncalabrese) [inline static/js/client.js]: Codacy Issue found: [Expected '===' and instead saw '=='.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/761380259/issues/source?bid=2452379&fileBranchId=2452987#l972)

File: static/js/client.js (hunk 8 above)
Comment (jasoncalabrese) [inline static/js/client.js]: Codacy Issue found: ['arcs' is defined but never used.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/761380259/issues/source?bid=2452379&fileBranchId=2452987#l1211)

File: lib/sandbox.js (hunk 4 above)
Comment (jasoncalabrese) [inline lib/sandbox.js]: Codacy Issue found: [Expected '===' and instead saw '=='.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/761380252/issues/source?bid=2452379&fileBranchId=2452987#l165)

File: static/js/client.js [hunk 9]
```diff
@@ -493,14 +498,14 @@ var app = {}, browserSettings = {}, browserStorage = $.localStorage;
         })
         .attr('fill', function (d) { return d.color; })
         .attr('opacity', function (d) { return futureOpacity(d.date.getTime() - latestSGV.x); })
-        .attr('stroke-width', function (d) { if (d.type == 'mbg') return 2; else return 0; })
+        .attr('stroke-width', function (d) { return d.type == 'mbg' ? 2 : 0; })
```
Comment (jasoncalabrese) [inline static/js/client.js]: Codacy Issue found: [Expected '===' and instead saw '=='.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/761380259/issues/source?bid=2452379&fileBranchId=2452987#l501)

File: static/js/client.js (hunk 7 above)
Comment (jasoncalabrese) [inline static/js/client.js]: Codacy Issue found: [Expected '===' and instead saw '=='.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/762740348/issues/source?bid=2452379&fileBranchId=2452987#l972)

File: static/js/client.js (hunk 8 above)
Comment (jasoncalabrese) [inline static/js/client.js]: Codacy Issue found: ['arcs' is defined but never used.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/762740348/issues/source?bid=2452379&fileBranchId=2452987#l1211)

File: lib/sandbox.js (hunk 4 above)
Comment (jasoncalabrese) [inline lib/sandbox.js]: Codacy Issue found: [Expected '===' and instead saw '=='.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/762740341/issues/source?bid=2452379&fileBranchId=2452987#l165)

File: static/js/client.js (hunk 9 above)
Comment (jasoncalabrese) [inline static/js/client.js]: Codacy Issue found: [Expected '===' and instead saw '=='.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/762740348/issues/source?bid=2452379&fileBranchId=2452987#l501)

File: static/js/client.js (hunk 7 above)
Comment (jasoncalabrese) [inline static/js/client.js]: Codacy Issue found: [Expected '===' and instead saw '=='.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/762782163/issues/source?bid=2452379&fileBranchId=2452987#l972)

File: static/js/client.js (hunk 8 above)
Comment (jasoncalabrese) [inline static/js/client.js]: Codacy Issue found: ['arcs' is defined but never used.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/762782163/issues/source?bid=2452379&fileBranchId=2452987#l1211)

File: lib/sandbox.js (hunk 4 above)
Comment (jasoncalabrese) [inline lib/sandbox.js]: Codacy Issue found: [Expected '===' and instead saw '=='.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/762782156/issues/source?bid=2452379&fileBranchId=2452987#l165)

File: static/js/client.js (hunk 9 above)
Comment (jasoncalabrese) [inline static/js/client.js]: Codacy Issue found: [Expected '===' and instead saw '=='.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/762782163/issues/source?bid=2452379&fileBranchId=2452987#l501)

File: static/js/client.js [hunk 10]
```diff
@@ -1199,8 +1208,8 @@ var app = {}, browserSettings = {}, browserStorage = $.localStorage;
       });
     var arcs = treatmentDots.append('path')
```
Comment (jasoncalabrese) [inline static/js/client.js]: Codacy Issue found: ['arcs' is defined but never used.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/763992950/issues/source?bid=2452379&fileBranchId=2452987#l1209)

File: static/js/client.js [hunk 11]
```diff
@@ -1290,9 +1302,10 @@ var app = {}, browserSettings = {}, browserStorage = $.localStorage;
       };
       predicted.forEach(function (d) {
         d.type = 'forecast';
-        if (d.sgv < BG_MIN)
+        if (d.sgv < BG_MIN) {
           d.color = 'transparent';
-      })
+        }
+      });
```
Comment (jasoncalabrese) [inline static/js/client.js]: Codacy Issue found: [Don't make functions within a loop.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/763992950/issues/source?bid=2452379&fileBranchId=2452987#l1308)

**Pull request merged**